transcom/mymove
# B-19120

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19120)

## Summary

This PR fixes some issues with TIO payment request weight review.
1. PPMs are no longer part of the weight review process
2. In the weight review process, "Total billable weight" now reads "Actual billable weight"
3. Under "Actual billable weight" the shipments are now labeled with shipment type, with HHGs numbered if there is more than 1 HHG
4. Weight requested in weight review process no longer incorporates PPM weight
5. 110% of total estimated weight no longer includes PPMs and their estimated weight

### How to test

1. Access the application as a TIO
2. Select moves, click Payment requests, and find ones that will help you confirm the above changes:
- a move with a PPM
  - Click Review weights and confirm that the weight requested does not include PPM weight (4), and that "Actual billable weight" now appears instead of "Total billable weight" (2)
  - Click Edit and confirm that the "110% of total estimated weight" no longer includes PPM weight(s) (5)
  - Click Review shipment weights and confirm that the PPM no longer appears for review (1)
- a move with multiple HHGs
  - Click Review weights and confirm that HHGs are now labeled under "Actual billable weight" and are numbered (3)
- a move with NTS/NTS-R
  - Click Review weights and confirm that NRS/NTS-Rs are now labeled under "Actual billable weight" (3)
